crates/bitwarden-crypto/src/keys/fingerprint.rs

```diff
@@ -0,0 +1,113 @@
+//! This module provides functionality to generate a cryptographic fingerprint for a public key.
```

Comment:
Note, this duplicates the notion of fingerprints; there is also bitwarden-crypto/src/fingerprint.rs. Suggestion: Rename the latter to "fingerprint-phrase" or "human-readable-fingerprint" - which is the mechanism to get a human-readable phrase from a canonical byte array (be that the public key DER; or in the future the cryptographic fingerprint (derived with the new fingerprint.rs) from the verifying key).

Reply:
Or AccountFingerprint.